source/module_orbital/CMakeLists.txt

```diff
@@ -13,3 +13,8 @@ add_library(
     ORB_table_beta.cpp
     ORB_table_phi.cpp
 )
+
+IF (BUILD_TESTING)
+  set(CMAKE_CXX_STANDARD 14)
```

Comment:
Would you move this line above to 
https://github.com/deepmodeling/abacus-develop/blob/b6e9a7ec60d3a9106b25ab7da947508ce218ded9/CMakeLists.txt#L233